upsolver-ui/src/components/App/App.jsx

```diff
@@ -21,6 +22,31 @@ export default function App() {
     localStorage.setItem("user", JSON.stringify(user));
   }, [user]);
 
+  useEffect(() => {
+    async function deletUserIfNotLoggedIn() {
+      try {
+        const response = await fetch(
+          "http://localhost:3001/users/is-logged-in",
+          {
+            method: "GET",
+            headers: {
+              "Content-Type": "application/json",
+              "x-access-token": user?.token,
+            },
+          }
+        );
+        const data = await response.json();
+        console.log("isLoggedIn response: ", data);
+        if (data.isLoggedIn === false) {
+          updateUser(null);
+        }
+      } catch (err) {
+        console.error("[!] Unable to verify login status", err.message);
```

Comment:
may be planned in a further PR, but we should make the UI tell the user that something went wrong as well instead of logging the error (again it's ok to log if in dev mode)

Reply:
Fixed in: https://github.com/cdamezcua/Upsolver/pull/21